### PR TITLE
Use integral types fmt specifiers from cinttypes header

### PR DESCRIPTION
**Pull request by andreas-bok-sociomantic**
The PRIxN adds support for width-based integral types
See more: http://www.cplusplus.com/reference/cinttypes

### CODE REVIEW

File: libdrizzle/binlog.cc [hunk 1]
```diff
@@ -493,9 +493,10 @@ void drizzle_binlog_get_filename(drizzle_st *con, char **filename, int file_inde
                             row_count - 1 : file_index;
 
     drizzle_row_t row = drizzle_row_index(result, row_index);
-    *filename =( char*)realloc(*filename, strlen(row[0]));
-    memcpy(*filename, row[0], strlen(row[0]));
-    *filename[strlen(row[0])] = '\0';
+    size_t len = strlen(row[0]);
+    *filename = (char*)realloc(*filename, len + 1);
+    memcpy(*filename, row[0], len);
+    (*filename)[len] = '\0';
```
Comment (ben-palmer-sociomantic) [inline libdrizzle/binlog.cc]: Hasn't this already been commited?

File: configure.ac [hunk 2]
```diff
@@ -6,7 +6,7 @@
 # Use and distribution licensed under the BSD license.  See
 # the COPYING file in this directory for full text.
 
-AC_INIT([libdrizzle-redux],[5.4.1],[https://github.com/sociomantic-tsunami/libdrizzle-redux/issues],[libdrizzle-redux],[https://github.com/sociomantic-tsunami/libdrizzle-redux])
+AC_INIT([libdrizzle-redux],[5.4.2],[https://github.com/sociomantic-tsunami/libdrizzle-redux/issues],[libdrizzle-redux],[https://github.com/sociomantic-tsunami/libdrizzle-redux])
```
Comment (ben-palmer-sociomantic) [inline configure.ac]: This has also already been merged. Are you sure this is rebased on master?

**Review by ben-palmer-sociomantic — COMMENTED** [review]
Waiting until the rebasing issues are sorted to continue my review.

**Comment by andreas-bok-sociomantic** [comment]
Ah true, hadn't rebased it on the latest v5.x.x branch

**Comment by ben-palmer-sociomantic** [comment]
This seems like it might be more of a new feature than a bug fix and so might belong in `v5.5.0`

**Comment by andreas-bok-sociomantic** [comment]
Changed milestone to `v5.5.0`

File: libdrizzle/handshake.cc [hunk 3]
```diff
@@ -445,7 +445,7 @@ drizzle_return_t drizzle_state_handshake_client_read(drizzle_st *con)
     if (con->packet_size != real_size)
     {
       drizzle_set_error(con, __func__,
-                        "bad packet size:%zu:%zu", real_size, con->packet_size);
+                        "bad packet size:%" PRIu64 ":%", real_size, con->packet_size);
```
Comment (ben-palmer-sociomantic) [inline libdrizzle/handshake.cc]: Is this line missing a format specifier?

**Review by ben-palmer-sociomantic — COMMENTED** [review]
One minor comment otherwise LGTM

**Comment by andreas-bok-sociomantic** [comment]
Fixed missing fmt specifier

```diff
diff --git a/libdrizzle/handshake.cc b/libdrizzle/handshake.cc
index 637e3d3..d7477c5 100644
--- a/libdrizzle/handshake.cc
+++ b/libdrizzle/handshake.cc
@@ -445,7 +445,8 @@ drizzle_return_t drizzle_state_handshake_client_read(drizzle_st *con)
     if (con->packet_size != real_size)
     {
       drizzle_set_error(con, __func__,
-                        "bad packet size:%" PRIu64 ":%", real_size, con->packet_size);
+                        "bad packet size:%" PRIu64 ":%" PRIu64,
+                        real_size, con->packet_size);
       return DRIZZLE_RETURN_BAD_HANDSHAKE_PACKET;
     }
 
```

**Comment by ben-palmer-sociomantic** [comment]
LGTM

**Pull request merged**